README.md

```diff
@@ -1,8 +1,9 @@
 # Horde.jl
+[![Build Status](https://travis-ci.com/mkschleg/Horde.jl.svg?branch=master)](https://travis-ci.com/mkschleg/Reproduce.jl)
```

Comment:
Looks like these might have the wrong link. I think the on-click event would lead to `Reproduce.jl`

Reply:
Oops. Right. Odd.

Reply:
I'm not quite sure how that happened.